testing/Makefile

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 #
```

Comment:
Can we get a file level comment saying what this is supposed to build?  Also, your header block here says "Copyright the Kubernetes Authors", which seems, uh, not right?

Reply:
Done